### PR TITLE
Fix NuGet/Home#776: MultipartWebRequest::CreateMultipartRequest should us "\r\n" explicitly and not Environment.NewLine

### PR DESCRIPTION
**Pull request by feiling**
A simple fix.

@emgarten @yishaigalatzer @zhili1208 @deepakaravindr

### CODE REVIEW

File: src/Core/Http/MultipartWebRequest.cs [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void CreateMultipartRequest(WebRequest request)
                     stream.Write(headerBytes, 0, headerBytes.Length);
                 }
 
-                byte[] newlineBytes = Encoding.UTF8.GetBytes(Environment.NewLine);
+                byte[] newlineBytes = Encoding.UTF8.GetBytes("\r\n");
```
Comment (emgarten) [inline src/Core/Http/MultipartWebRequest.cs:63]: Make this line ending a const string and use that in both places.

**Comment by emgarten** [comment]
:shipit:

**Comment by yishaigalatzer** [comment]
:shipit:

**Comment by feiling** [comment]
updated per Justin's comment. Rebased & merged.

**Pull request closed without merging**